flixel/system/ui/FlxSoundTray.hx

```diff
@@ -108,18 +108,18 @@ class FlxSoundTray extends Sprite
 	}
 
 	/**
-	 * This function just updates the soundtray object.
+	 * This function updates the soundtray object.
 	 */
 	public function update(MS:Float):Void
 	{
-		// Animate stupid sound tray thing
+		// Animate sound tray thing
 		if (_timer > 0)
 		{
-			_timer -= MS / 1000;
+			_timer -= (MS / 1000);
 		}
 		else if (y > -height)
 		{
-			y -= (MS / 1000) * FlxG.height * 2;
+			y -= FlxG.height * FlxG.elapsed;
```

Comment:
This function is called directly from an openfl onEnterframe event listener, not Flixel's update loop, meaning it should use MS/1000 rather than FlxG.elapsed. this change does not "untie" the sound tray's animation from frameRate, as the title says, instead it is now using the (potentially) fixed update elapsed value when its actually being called from a variable, or non-fixed elapse value

Reply:
side note: I think using `FlxG.height` here is a typo. I believe it was supposed to be the following, which is much, much smoother
```suggestion
			y -= (MS / 1000) * height * 2;
```